packages/wallet/src/index.ts

```diff
@@ -16,44 +16,43 @@ import { pbkdf2 } from 'ethereum-cryptography/pbkdf2'
 import { getRandomBytesSync } from 'ethereum-cryptography/random'
 import { scrypt } from 'ethereum-cryptography/scrypt'
 import { sha256 } from 'ethereum-cryptography/sha256'
-
-import { bufferToHex } from './util/bytes'
+import { concatBytes, equalsBytes, hexToBytes, utf8ToBytes } from 'ethereum-cryptography/utils'
 
 export { EthereumHDKey as hdkey } from './hdkey'
 export { Thirdparty as thirdparty } from './thirdparty'
 const uuidv4 = require('uuid').v4
 
 const bs58check = base58check(sha256)
 function randomBytes(num: number) {
```

Comment:
The randombytes method could be refactored to use the one from utils (same implementation)